From david-b@pacbell.net Tue Dec 12 17:24:16 2006
From: David Brownell <david-b@pacbell.net>
To: Greg KH <greg@kroah.com>,
 linux-usb-devel@lists.sourceforge.net
Subject: USB: define USB_CLASS_MISC in <linux/usb/ch9.h>
Date: Tue, 12 Dec 2006 15:12:30 -0800
MIME-Version: 1.0
Content-Type: text/plain;
  charset="us-ascii"
Content-Disposition: inline
Message-Id: <200612121512.30977.david-b@pacbell.net>

Add USB_CLASS_MISC to <linux/usb/ch9.h>

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/ch9.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/usb/ch9.h
+++ gregkh-2.6/include/linux/usb/ch9.h
@@ -224,6 +224,7 @@ struct usb_device_descriptor {
 #define USB_CLASS_CONTENT_SEC		0x0d	/* content security */
 #define USB_CLASS_VIDEO			0x0e
 #define USB_CLASS_WIRELESS_CONTROLLER	0xe0
+#define USB_CLASS_MISC			0xef
 #define USB_CLASS_APP_SPEC		0xfe
 #define USB_CLASS_VENDOR_SPEC		0xff
 
